prow/proxy-presubmit.sh

```diff
@@ -29,24 +29,10 @@ set -u
 # Print commands
 set -x
 
-if [ "${CI:-}" == 'bootstrap' ]; then
-  # Test harness will checkout code to directory $GOPATH/src/github.com/istio/proxy
-  # but we depend on being at path $GOPATH/src/istio.io/proxy for imports
-  ln -sf ${GOPATH}/src/github.com/istio ${GOPATH}/src/istio.io
-  ROOT=${GOPATH}/src/istio.io/proxy
-  cd ${GOPATH}/src/istio.io/proxy
-
-  # Use the provided pull head sha, from prow.
-  GIT_SHA="${PULL_PULL_SHA}"
-else
-  # Remove old bazel.rc.ci
-  rm -f "${HOME}/.bazelrc"
-
-  # Use the current commit.
-  GIT_SHA="$(git rev-parse --verify HEAD)"
-fi
-
-cd $ROOT
+GOPATH=/go
+ROOT=/go/src
+# Remove old bazel.rc.ci
```

Comment:
nit: fix this comment?

Reply:
it's from line 43 in old file.

Reply:
I understand, but I thought it'd be good to fix.